chaossearch/resource_object_group.go

```diff
@@ -77,6 +77,16 @@ func resourceObjectGroup() *schema.Resource {
 				Optional:    true,
 				ForceNew:    false,
 			},
+			"column_renames": {
```

Comment:
Just a matter of taste, I'd put this one above `index_retention` block, because index_retention has `forceNew: false` and all other fields have it `true`